### ISSUE TITLE
Disable warning is spamming console

### ISSUE THREAD

**Issue by reitzmichnicht** [qoomon/maven-git-versioning-extension#61]
I need to disable the versioning for some maven goals, this results in lots of warnings:
```
$ mvn -B release:prepare -DdryRun=true -Dversioning.disable=true -DautoVersionSubmodules=true
[INFO] Scanning for projects...
[INFO]
[INFO] ----------------- maven-git-versioning-extension:4.10.1 ----------------
[INFO] Adjusting project models...
[INFO]
[WARNING] skip - versioning is disabled
[WARNING] skip - versioning is disabled
[WARNING] skip - versioning is disabled
[WARNING] skip - versioning is disabled
[WARNING] skip - versioning is disabled
[WARNING] skip - versioning is disabled
[WARNING] skip - versioning is disabled
[WARNING] skip - versioning is disabled
[WARNING] skip - versioning is disabled
[WARNING] skip - versioning is disabled
[WARNING] skip - versioning is disabled
[WARNING] skip - versioning is disabled
[WARNING] skip - versioning is disabled
[WARNING] skip - versioning is disabled
[WARNING] skip - versioning is disabled
[WARNING] skip - versioning is disabled
[WARNING] skip - versioning is disabled
[WARNING] skip - versioning is disabled
[WARNING] skip - versioning is disabled
[WARNING] skip - versioning is disabled
[WARNING] skip - versioning is disabled
[WARNING] skip - versioning is disabled
[WARNING] skip - versioning is disabled
[WARNING] skip - versioning is disabled
[WARNING] skip - versioning is disabled
[WARNING] skip - versioning is disabled
[WARNING] skip - versioning is disabled
[WARNING] skip - versioning is disabled
[WARNING] skip - versioning is disabled
[WARNING] skip - versioning is disabled
[WARNING] skip - versioning is disabled
[WARNING] skip - versioning is disabled
[WARNING] skip - versioning is disabled
[WARNING] skip - versioning is disabled
[WARNING] skip - versioning is disabled
[WARNING] skip - versioning is disabled
[WARNING] skip - versioning is disabled
[WARNING] skip - versioning is disabled
[WARNING] skip - versioning is disabled
[WARNING] skip - versioning is disabled
[WARNING] skip - versioning is disabled
[WARNING] skip - versioning is disabled
[WARNING] skip - versioning is disabled
[WARNING] skip - versioning is disabled
[WARNING] skip - versioning is disabled
[WARNING] skip - versioning is disabled
[WARNING] skip - versioning is disabled
[WARNING] skip - versioning is disabled
[INFO] Reactor Build Order:
[INFO]
[INFO] FOO Parent pom                                              [pom]
[INFO] FOO rules                                                   [jar]
[INFO] FOO unit test depchain                                      [pom]
[INFO] FOO integration test depchain                               [pom]
[WARNING] skip - versioning is disabled
[WARNING] skip - versioning is disabled
[WARNING] skip - versioning is disabled
[WARNING] skip - versioning is disabled
[WARNING] skip - versioning is disabled
[WARNING] skip - versioning is disabled
[WARNING] skip - versioning is disabled
[WARNING] skip - versioning is disabled
[WARNING] skip - versioning is disabled
[WARNING] skip - versioning is disabled
[WARNING] skip - versioning is disabled
[WARNING] skip - versioning is disabled
[WARNING] skip - versioning is disabled
[WARNING] skip - versioning is disabled
[WARNING] skip - versioning is disabled
[WARNING] skip - versioning is disabled
[WARNING] skip - versioning is disabled
[WARNING] skip - versioning is disabled
[WARNING] skip - versioning is disabled
[WARNING] skip - versioning is disabled
[WARNING] skip - versioning is disabled
[WARNING] skip - versioning is disabled
[WARNING] skip - versioning is disabled
[WARNING] skip - versioning is disabled
[WARNING] skip - versioning is disabled
[WARNING] skip - versioning is disabled
[WARNING] skip - versioning is disabled
[WARNING] skip - versioning is disabled
[WARNING] skip - versioning is disabled
[WARNING] skip - versioning is disabled
[WARNING] skip - versioning is disabled
[WARNING] skip - versioning is disabled
[WARNING] skip - versioning is disabled
[WARNING] skip - versioning is disabled
[WARNING] skip - versioning is disabled
[WARNING] skip - versioning is disabled
[WARNING] skip - versioning is disabled
[WARNING] skip - versioning is disabled
[WARNING] skip - versioning is disabled
[WARNING] skip - versioning is disabled
[WARNING] skip - versioning is disabled
[WARNING] skip - versioning is disabled
[WARNING] skip - versioning is disabled
[WARNING] skip - versioning is disabled
[WARNING] skip - versioning is disabled
[WARNING] skip - versioning is disabled
[WARNING] skip - versioning is disabled
[WARNING] skip - versioning is disabled
[WARNING] skip - versioning is disabled
[WARNING] skip - versioning is disabled
[WARNING] skip - versioning is disabled
[WARNING] skip - versioning is disabled
[WARNING] skip - versioning is disabled
[WARNING] skip - versioning is disabled
[WARNING] skip - versioning is disabled
[WARNING] skip - versioning is disabled
[WARNING] skip - versioning is disabled
[WARNING] skip - versioning is disabled
[WARNING] skip - versioning is disabled
[WARNING] skip - versioning is disabled
[WARNING] skip - versioning is disabled
[WARNING] skip - versioning is disabled
[WARNING] skip - versioning is disabled
[WARNING] skip - versioning is disabled
[WARNING] skip - versioning is disabled
[WARNING] skip - versioning is disabled
[WARNING] skip - versioning is disabled
[WARNING] skip - versioning is disabled
[WARNING] skip - versioning is disabled
[WARNING] skip - versioning is disabled
[WARNING] skip - versioning is disabled
[WARNING] skip - versioning is disabled
[WARNING] skip - versioning is disabled
[WARNING] skip - versioning is disabled
[WARNING] skip - versioning is disabled
[WARNING] skip - versioning is disabled
[WARNING] skip - versioning is disabled
[WARNING] skip - versioning is disabled
[WARNING] skip - versioning is disabled
[WARNING] skip - versioning is disabled
[WARNING] skip - versioning is disabled
[WARNING] skip - versioning is disabled
[WARNING] skip - versioning is disabled
[WARNING] skip - versioning is disabled
[WARNING] skip - versioning is disabled
[WARNING] skip - versioning is disabled
[WARNING] skip - versioning is disabled
[WARNING] skip - versioning is disabled
[WARNING] skip - versioning is disabled

```
As the disabling is intended, it should be a single INFO message, not hundreds of warnings.

**Comment by qoomon** [comment]
:-D  I totally agree 👍

**Comment by qoomon** [comment]
it is fixed will be part of the next release

**Comment by qoomon** [comment]
fixed within version `4.10.2`

**Comment by reitzmichnicht** [comment]
Thanks a lot, works as expected now. Little PR #64 as improvement.
